src/Block/AdminListBlockService.php

```diff
@@ -51,13 +51,13 @@ public function __construct(
         Pool $pool,
         ?TemplateRegistryInterface $templateRegistry = null
     ) {
-        parent::__construct($twigOrName, $templating);
+        parent::__construct($twigOrName);
 
         $this->pool = $pool;
         $this->templateRegistry = $templateRegistry ?: new TemplateRegistry();
     }
 
-    public function execute(BlockContextInterface $blockContext, ?Response $response = null)
+    public function execute(BlockContextInterface $blockContext, ?Response $response = null): Response
```

Comment:
Unfortunately, adding typehint is a BC-break...

Reply:
If those 4 typehints are the only thing that prevents the admin to run block bundle 4, maybe it is something to think if we want to break bc here, just to move forward... wdyt?

Reply:
An upgrade note will be required. But I agree we should accept an exception.